src/components/ha-card.ts

```diff
@@ -27,24 +28,46 @@ class HaCard extends LitElement {
         color: var(--primary-text-color);
         display: block;
         transition: all 0.3s ease-out;
+        position: relative;
       }
-      .header:not(:empty),
-      .header::slotted(*) {
+
+      :host ::slotted(*) {
+        display: block;
```

Comment:
I think that it's weird that we put `display: block` on all elements that get slotted ?